astropy/coordinates/builtin_frames/__init__.py

```diff
@@ -88,8 +88,8 @@ def make_transform_graph_docs(transform_graph):
                                             priorities=False)
 
     docstr = """
-    The diagram below shows all of the coordinate systems built into the
-    `~astropy.coordinates` package, their aliases (useful for converting
+    The diagram below shows all of the built in coordinate systems,
+    their aliases (useful for converting
```

Comment:
Could you change the wrapping to the end, so that things are better aligned (i.e., ESC-q in emacs).